CODE_OF_CONDUCT.md

```diff
@@ -2,13 +2,13 @@
 
 ## 1. Purpose
 
-A primary goal of the UI-Kit is to be inclusive to the largest number of contributors, with the most varied and diverse backgrounds possible. As such, we are
+A primary goal of the components is to be inclusive to the largest number of contributors, with the most varied and diverse backgrounds possible. As such, we are
```

Comment:
A primary goal of the  `design-system-components` is to be inclusive to the largest number of contributors, with the most varied and diverse backgrounds possible.